project_task_code/models/project_task.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2016 Tecnativa <vicent.cubells@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class ProjectTask(models.Model):
+    _inherit = "project.task"
+
+    code = fields.Char(
+        string="Task Number",
+        required=True,
+        default="/",
+        readonly=True,
```

Comment:
Not for migration: What is the reason behind keeping this field readonly?

Reply:
Because it is generated through a sequence :-)